ConvertToJpg/ConvertToJpg.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.8.34330.188
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ConvertToJpg", "ConvertToJpg.csproj", "{8F94372B-833A-40F5-AFEC-89BCB33B9E9F}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|x64 = Debug|x64
+		Release|x64 = Release|x64
```

Comment:
When I build and run the sample with these changes, it builds and runs in Debug. Could we set the default configuration to be Release?

Reply:
The .sln file was removed because it wasn't necessary to run the sample, so this comment can be disregarded.